packages/forms/backend/sirius-components-collaborative-forms/src/main/resources/schema/form.graphqls

```diff
@@ -221,6 +221,26 @@ type Link implements Widget {
   url: String!
 }
 
+type Button implements Widget {
+  id: ID!
+  diagnostics: [Diagnostic!]!
+  label: String!
+  buttonLabel: String
+  imageURL: String
+  iconURL: String
+  style: ButtonStyle
+}
+
+type ButtonStyle {
+  backgroundColor: String
+  foregroundColor: String
+  fontSize: Int
+  italic: Boolean
+  bold: Boolean
+  underline: Boolean
+  strikeThrough: Boolean
+}
+
```

Comment:
Am I missing something or this cannot possibly work since the schema does not declare the mutation, the input and payload necessary to perform the pushButton mutation?

Reply:
```
Query did not validate : 'mutation pushButton($input: PushButtonInput!) {
  pushButton(input: $input) {
    __typename
    ... on ErrorPayload {
      message
      __typename
    }
  }
}'
```

As I suspected it, the code cannot work.